image:
  resize:
    width: 480
    height: 270
  # 0 is a top of raw_img and 100 is a bottom of raw_img
  roi:
    top_location: 30
    bottom_location: 80
  # binary threshold
  binary_thres:
    gray: 180
    hsv_s: 150